tests/link/jax/test_subtensor.py

```diff
@@ -13,54 +15,62 @@
 
 
 def test_jax_Subtensor_constant():
+    shape = (3, 4, 5)
+    x_pt = tensor("x", shape=shape, dtype="int")
+    x_np = np.arange(np.prod(shape)).reshape(shape)
+
     # Basic indices
-    x_pt = pt.as_tensor(np.arange(3 * 4 * 5).reshape((3, 4, 5)))
     out_pt = x_pt[1, 2, 0]
     assert isinstance(out_pt.owner.op, pt_subtensor.Subtensor)
-    out_fg = FunctionGraph([], [out_pt])
-    compare_jax_and_py(out_fg, [])
+    out_fg = FunctionGraph([x_pt], [out_pt])
```

Comment:
Otherwise we were probably never testing jax Path as the index operation would be just constant_folded. This was not always the case, as we used to not run any rewrites in `compare_jax_and_py`. Now we do